### PR TITLE
conf: split usage of registry_uri and openshift_uri

### LINKED ISSUE #259: Configuration: some values are used on multiple places
...which makes testing (and some deployments) pretty much impossible. E.g. `openshift_uri`, `registry`. We should be able to override/change those for each specific usage

### PR DESCRIPTION
**Pull request by TomasTomecek**
* rename openshift_uri -> openshift_url (with deprecation warning)

Fixes #259

### CODE REVIEW

File: osbs/conf.py [hunk 1]
```diff
@@ -270,6 +280,10 @@ def get_builder_use_auth(self):
                                default=self.get_use_auth(),
                                is_bool_val=True)
 
+    def get_builder_openshift_url(self):
+        """ url of OpenShift where builder will connect """
+        return self._get_value("builder_openshift_url", self.conf_section, "builder_openshift_url")
```
Comment (mmilata) [inline osbs/conf.py]: Should we perhaps fall back to the value of `openshift_url` if `builder_openshift_url` is not defined?

**Comment by mmilata** [comment]
Can you please document the changes in `docs/configuration_file.md`?

**Comment by TomasTomecek** [comment]
@mmilata I knew I forgot something! Thanks

**Comment by TomasTomecek** [comment]
done

File: osbs/conf.py (hunk 1 above)
Comment (TomasTomecek) [inline osbs/conf.py]: Okay, I think that's reasonable.

**Comment by TomasTomecek** [comment]
fallback comment resolved

**Comment by mmilata** [comment]
Merged, thanks!

**Pull request merged**